### ISSUE TITLE
Change landing page to dynamic cms as discussed

### ISSUE THREAD

**Issue by sherif807** [sherif807/information_landing#2]
@MonikaBi 

after this person sends the react application for the page, I will give you a react developer that will change the page to be dynamic, we basically need the following so you can explain to him, and possibly check his work. 

1- ability to change the logo, the image and the main wording and the minor 

![image](https://user-images.githubusercontent.com/8910175/195518962-d14fc211-6910-4b3c-8f1d-c835e2fdf453.png)

2- ability to change the paragraphs, add up to 6 paragraphs.

![image](https://user-images.githubusercontent.com/8910175/195655456-e6c9bd4b-9de9-4508-b80a-abd3ed10bb15.png)

3- the pricing 

![image](https://user-images.githubusercontent.com/8910175/195519169-e8bff0e2-259d-4e45-be3f-637688260e30.png)

4- Contact us, make it work

![image](https://user-images.githubusercontent.com/8910175/195519252-8fdf62e0-0e91-4037-bac1-7beb477a69c1.png)

5- Make an admin page with login

basically he needs to give us a very quick admin page, that we can manually edit these and will be added to a database, then will move on to automating google ads, 

so again the plan

1- ability to login and edit the page being advertised
2- with a click of a button, Generate a google ad, send it to google, start advertising for a price
3- collect information about the idea, is it successful or its not? 

Thank you

**Comment by MonikaBi** [comment]
Hi @sherif807! Thanks for adding me to this project. 
As you mentioned this is the project space with Mahmoud, I just wanted to clarify if I still need to explain what is needed? 

Let me know - thank you :)

**Comment by sherif807** [comment]
@MonikaBi 

Just be here for you can review the work and guide it, will this be ok?

**Comment by MonikaBi** [comment]
@sherif807 Of course! Just wanted to make sure :)
